server/axios_api.js

```diff
@@ -0,0 +1,62 @@
+var axios = require("axios").default;
+var HOST = "movies-tvshows-data-imdb.p.rapidapi.com";
+var API_KEY = "030611f594msh73349b7b00af6e8p16d57ejsncbf1b45a6165";
+var URL = "https://movies-tvshows-data-imdb.p.rapidapi.com/";
+
+
+
+function get_details(id) {
+  var options = {
+    method: 'GET',
+    url: URL,
+    params: {type: 'get-movie-details', imdb: id},
+    headers: {
+      'x-rapidapi-host': HOST,
+      'x-rapidapi-key': API_KEY
+    }
+  };
+  axios.request(options).then(function (response) {
+    console.log("rating:"+response.data["imdb_rating"]+"\n"+"popularity:"+response.data["popularity"]+"\n"+"vote:"+response.data["vote_count"]);
```

Comment:
looks good. In the end we need this function to return us object like below:
{
  imdbRating: <some_value>,
  rottenTomatoRating: <some_value>
}

where some_value value will be either the rating given by API or word 'N/A'. This is because for some movies ratings might not be available. In those cases we can show N/A instead of undefined in the UI

Reply:
ok, that should be easy to just return the rating

Reply:
yeah we shall return an object with data what client is supposed to consume. If a particular rating is not available then backend can send N/A instead for that